.github/workflows/build.yml

```diff
@@ -14,13 +14,25 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-24.04
+    runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
       - name: Setup .NET 9.0
         uses: actions/setup-dotnet@v2
         with:
           dotnet-version: 9.0.x
+      - name: Setup .NET 8.0
+        uses: actions/setup-dotnet@v2
+        with:
+          dotnet-version: 8.0.x
+      - name: Setup .NET 6.0
+        uses: actions/setup-dotnet@v2
+        with:
+          dotnet-version: 6.0.x
+      - name: Setup .NET 3.1
+        uses: actions/setup-dotnet@v2
+        with:
+          dotnet-version: 3.1.x
```

Comment:
I'm not getting how this extra SDKs are used. could you shed a light? I don't see a multi-build process/pipeline, but maybe I'm missing something

Reply:
The multi build is triggered by the multiple frameworks declared in the test project, I was under the impression that using the latest sdk could also build previous versions of net but that doesn't seem the case in our ci

Reply:
Ok cool! Thanks!